backend/sources/facebook/connector/src/main/resources/application.properties

```
@@ -2,3 +2,5 @@ kafka.brokers=${KAFKA_BROKERS}
 kafka.schema-registry-url=${KAFKA_SCHEMA_REGISTRY_URL}
 kafka.cleanup=${KAFKA_CLEANUP:false}
 kafka.commit-interval-ms=${KAFKA_COMMIT_INTERVAL_MS:30000}
+
+kafka.suppress-interval-ms=3000
```

```suggestion
kafka.suppress-interval-ms={KAFKA_SUPPRESS_INTERVAL_MS:3000}
```

may be a bit more future proof